File: FOLORUNSHO OLAMIDE/javaScript/CheckOutApp.js
const prompt = require("prompt-sync")();

    let itemList = [];
    let unitList = [];
    let piecesList = [];

    let name = prompt("What is your full name:\n ");

    let addItem = "YES";
    let pieces = 0;
    let unit = 0;

    while (addItem !== "NO") {
        let product = prompt("What did the user buy?:\n ");

        pieces = parseFloat(prompt("How many pieces?:\n "));
        unit = parseFloat(prompt("How much per unit?:\n "));

        itemList.push(product);
        unitList.push(unit);
        piecesList.push(pieces);

        addItem = prompt("Add more items? (Yes/No):\n ").toUpperCase();
    }

    let subTotal = 1;
    let discount = parseFloat(prompt("Enter discount (%): "));
    let discountAmount = (discount / 100) * subTotal;
    let finalTotal = subTotal - discountAmount;

    console.log("\n\n SEMICOLON STORES");
    console.log(" MAIN BRANCH");
    console.log(" LOCATION: 312, HERBERT MACAULAY WAY,  SABO YABA, LAGOS.");
    console.log(" TEL: 08123425776 ");
    console.log(" Date:", new Date().toLocaleDateString());
    console.log(" Cashier: Adam ");
    console.log(" Customer Name:", name);
    console.log("================================================================");
    console.log(" ITEM\t\tQTY\tPRICE\tTOTAL(NGN)");
    console.log("----------------------------------------------------------------");

    for (let counter = 0; counter < unitList.length; counter++) {
        let itemTotal = unitList[counter] * piecesList[counter];
        subTotal += unitList[counter] * piecesList[counter];
        console.log(` ${itemList[counter]}\t\t${piecesList[counter]}\t${unitList[counter]}\t${itemTotal}`);
    }

    console.log("================================================================");
    console.log(" Sub Total:", subTotal);
    console.log(" Discount:", discountAmount);
    console.log(" Final Total:", finalTotal);
}

File: FOLORUNSHO OLAMIDE/java/CheckOutApp.py
from datetime import date


    itemList = []
    unitList = []
    piecesList = []

    name = input("What is your full name:\n ")

    addItem = "YES"
    pieces = 0
    unit = 0

    while addItem != "NO":
        product = input("What did the user buy?:\n ")

        pieces = float(input("How many pieces?:\n "))
        unit = float(input("How much per unit?:\n "))

        itemList.append(product)
        unitList.append(unit)
        piecesList.append(pieces)

        addItem = input("Add more items? (Yes/No):\n ").upper()

    subTotal = 1
    discount = float(input("Enter discount (%): "))
    discountAmount = (discount / 100) * subTotal
    finalTotal = subTotal - discountAmount

    print("\n\n SEMICOLON STORES")
    print(" MAIN BRANCH")
    print(" LOCATION: 312, HERBERT MACAULAY WAY,  SABO YABA, LAGOS.")
    print(" TEL: 08123425776 ")
    print(" Date:", date.today())
    print(" Cashier: Adam ")
    print(" Customer Name:", name)
    print("================================================================")
    print(" ITEM\t\tQTY\tPRICE\tTOTAL(NGN)")
    print("----------------------------------------------------------------")

    for counter in range(len(unitList)):
        itemTotal = unitList[counter] * piecesList[counter]
        subTotal += unitList[counter] * piecesList[counter]
        print(f" {itemList[counter]}\t\t{piecesList[counter]}\t{unitList[counter]}\t{itemTotal}")

    print("================================================================")
    print(" Sub Total:", subTotal)
    print(" Discount:", discountAmount)
    print(" Final Total:", finalTotal)
